test/basic/src/test/java/io/narayana/lra/arquillian/FailingParticipantCallsIT.java

```diff
@@ -53,8 +54,8 @@ public void testFailingAfterLRA() throws LRACallbackException {
 
         try {
             response = client.target(UriBuilder.fromUri(baseURL.toExternalForm())
-                .path(FailingAfterLRAListener.ROOT_PATH)
-                .path(FailingAfterLRAListener.ACTION_PATH).build())
+                    .path(FailingAfterLRAListener.ROOT_PATH)
```

Comment:
Mentioning that formatting changes are expected to be in a separate commit: https://github.com/jbosstm/lra/blob/main/CONTRIBUTING.md#coding-guidelines (but @marcosgopen should comment how strict that is for Narayana LRA project)

Reply:
I am not strict on the rule when there are no many reformatting changes. I generally appreciate having a clean code and possibly I would appreciate having the reformatting changes in a different commit to help the reviewer(s) focus on the functional changes. Thanks

Reply:
Will set up formatter plugin when I have a while.

Reply:
@marcosgopen maybe (if the Narayana LRA community is interested in it) - the code style thing could be ran as part of the maven build to help with identifying observations like this? Do you think that would be helpful?

Reply:
The checkstyle plugin is enabled (https://github.com/jbosstm/lra/blob/main/pom.xml#L677-L679) and format error will make the build fail according to our configuration (https://github.com/jbosstm/narayana-checkstyle-config/blob/main/src/main/resources/narayana-checkstyle/checkstyle.xml). The formatting change proposed by Martin is not included in the narayana-checkstyle-config's rules. cc @tomjenkinson

Reply:
yes, I want to add a plugin that will automatically reformat the code as part of the build.